src/gconfig/config.py

```diff
@@ -106,15 +106,15 @@ def get(
         required: bool = None,
         default: any = None,
     ) -> str:
```

Comment:
Everything that uses `get` returns optional but `get` doesn't, should we also add this typing here?

Reply:
get is only used internally, but we should, adding it